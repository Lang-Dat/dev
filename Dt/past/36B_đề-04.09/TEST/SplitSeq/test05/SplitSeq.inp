11
4 4
63926321 407246273 97480467 72915601
4 4
61882503 745945965 633260167 12043059
4 1
603441716 568791373 482478156 157335982
4 2
289938529 154971829 102106987 93675011
4 1
95529905 804005632 592361930 295057982
4 2
540587662 828853444 697476127 210897321
4 2
664790055 238240173 716410539 298465317
4 3
22904953 887019908 989883831 901888311
4 3
690899873 484741296 628687399 992612427
4 1
635947968 908499427 649760289 424604995
4 1
721938147 512232176 158753263 694556781
