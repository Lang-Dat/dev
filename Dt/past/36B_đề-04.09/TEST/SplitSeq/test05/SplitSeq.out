YES
1 2 3 4
YES
1 2 3 4
YES
4
YES
1 4
YES
4
YES
3 4
YES
1 4
YES
1 3 4
YES
1 3 4
YES
4
YES
4
